### PR TITLE
Answers -> Search 3: Replacing search-core imports

### PR DESCRIPTION
**Pull request by ElemelonWind**
Changed all `answers-core` deprecated imports to `search-core`

J=SLAP-2204
TEST=auto

### CODE REVIEW

**Review by yen-tt — COMMENTED** [review]
http-manager.ts comment still link to AnswersCore. Lets update that too

File: src/search-core-re-exports.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
 export {
   AdditionalHttpHeaders,
-  AnswersConfig,
```
[the remark this answers is not in the document]
Comment (ElemelonWind) [reply src/search-core-re-exports.ts:3]: Oh true, I'll add that back in

File: __mocks__/@yext/answers-core.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export * from '@yext/answers-core';
+export * from '@yext/search-core';
```
Comment (nmanu1) [inline __mocks__/@yext/answers-core.ts:1]: this file should be renamed

File: src/models/slices/meta.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Context } from '@yext/answers-core';
+import { Context } from '@yext/search-core';
 import { SearchType } from '../utils/searchType';
 
 /**
```
Comment (nmanu1) [inline src/models/slices/meta.ts:4]: the docs for `context` below reference "Answers"

**Review by nmanu1 — COMMENTED** [review]
have the `package.json` files in the `lib` directory been updated?

**Review by nmanu1 — APPROVED** [review]

**Pull request merged**